microcosm_pubsub/daemon.py

```diff
@@ -68,7 +71,24 @@ def defaults(self):
                     strategy_name=self.args.envelope,
                 ),
             )
-
+        # for AWS Lambda
+        # SQS message comes as function argument
+        # e.g
+        # ```
+        # def handler(event, context):
+        #     # processing event
+        # ```
+        # we don't use command line args here
+        if self.sqs_event:
+            config.update(
```

Comment:
I'm not a fan of this, but I can't say more since it's unclear how all of this will come together.